dbms/src/Storages/DeltaMerge/Index/MinMaxIndex.cpp

```diff
@@ -183,6 +183,17 @@ std::pair<Int64, Int64> MinMaxIndex::getIntMinMax(size_t pack_index)
     return {minmaxes->getInt(pack_index * 2), minmaxes->getInt(pack_index * 2 + 1)};
 }
 
+std::pair<std::string, std::string> MinMaxIndex::getIntMinMaxOrNull(size_t pack_index)
+{
+    std::string min, max;
+    Field minField, maxField;
+    minmaxes->get(pack_index * 2, minField);
+    minmaxes->get(pack_index * 2 + 1, maxField);
+    min = minField.isNull() ? "null" : minField.toString();
+    max = maxField.isNull() ? "null" : maxField.toString();
+    return {min, max};
+}
```

Comment:
I think it is just for debug, should make sure it was removed before merging to master.

Reply:
No problem